Azael/index.js

```diff
@@ -0,0 +1,24 @@
+// # 18-PB-arrays-math
+
+const colors = [
+  "red",
+  9999,
+  "green",
+  "blue",
+  777,
+  "yellow",
+  "black",
+  "purple",
+  "brown",
+];
+
+// *** Task // with Math.random() ==> get random item from colors array, if the item is number set it to ''
+
+let colorsLength = colors.length;
+
+console.log(colorsLength);
+
+let random = Math.floor(Math.random() * colorsLength);
+let randomColors = colors[random];
+
```

Comment:
let randomColorsNumber = randomColors.toString();

Reply:
You still get Numbers using this code. I'll share with you the best solution.
get rid of let randomColors and also randomColorsNumber and paste this code instead=>
let randomColorsNumber = typeof colors[random] === "number" ? "" : colors[random];